app/models/hyrax/file_metadata.rb

```diff
@@ -40,10 +40,9 @@ def uri_for(use:)
 
     # all remaining attributes are on AF::File metadata_node unless otherwise noted
     attribute :label, ::Valkyrie::Types::Set
-    attribute :original_filename, ::Valkyrie::Types::Set
+    attribute :original_filename, ::Valkyrie::Types::String
```

Comment:
A good bit of this was based off the Figgy FileMetadata class.  They had [original_filename defined as a Set](https://github.com/pulibrary/figgy/blob/main/app/resources/nested_resources/file_metadata.rb#L14).  It makes sense to me to have this be a String.  I don't know if @tpendragon can comment on why it is a Set in Figgy.